src/WorkBC.Data/Migrations/20240508173356_UpdateCareerProfilesWithNoc2021.cs

```diff
@@ -67,18 +69,19 @@ protected override void Down(MigrationBuilder migrationBuilder)
             //Reverting to the previous version of this SavedCareerProfiles table
             //Populate the values into EDM_CareerProfile_CareerProfileId column from Code2016 values of NocCodes2021 table.
             migrationBuilder.Sql(
-            @"Update [dbo].[SavedCareerProfiles]
-              Set EDM_CareerProfile_CareerProfileId = (
-              Select Code2016 from [dbo].NocCodes2021 nc 
-              where ((Select NocCodeId2021 from [dbo].[SavedCareerProfiles]) in (nc.Code))
-              )
+            @"Update SavedCareerProfiles
+              Set EDM_CareerProfile_CareerProfileId = Convert(int, Substring(nc.Code2016, 0, 5))
```

Comment:
I'm not sure this is correct, as per the `Up` method.

Reply:
In the Up, we have the choice to choose the top entry from the list of Noc2021 for the same multiple set of Noc2016 while populating "NocCodeId2021" column. We then proceed to delete the "EDM_CareerProfile_CareerProfileId" column as per the requirement.  In the Down, we have already deleted the EDM_CareerProfile_CareerProfileId column and so we have no way to trace which Noc2016 value is the original one for the Noc2021 column value. Thus, I populated it with the default first value.

Reply:
Makes sense, thanks.